### PR TITLE
feat(projects): add Vue InstantSearch

### PR DESCRIPTION
**Pull request by rayrutjes**
Hey, what is needed regarding the logo?
Any help would be appreciated.

### CODE REVIEW

File: src/algolia-projects.json [hunk 1]
```diff
@@ -76,6 +76,19 @@
...
   },
   {
+    "name": "Vue InstantSearch",
+    "category": "Library",
+    "description": "The Vue.js ultimate toolbox for creating instant-search experiences",
+    "url_home": "https://community.algolia.com/vue-instantsearch/",
+    "url_github": "https://github.com/algolia/vue-instantsearch",
+    "url_forum": "",
+    "discussion_count": "",
+    "icon": "instantsearch-react",
```
Comment (JonasBa) [inline src/algolia-projects.json]: Can you change icon : "vue-instantsearch" or "instantsearch-vue" and I'll add the icon :)?

Comment (rayrutjes) [inline src/algolia-projects.json]: Done

File: src/img/projects/instantsearch-vue.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 400" height="400" width="400"><path d="M237.417 86.655l-30.226 52.352-30.224-52.352H76.31l130.88 226.69 130.88-226.69z" fill="#4dba87"/><path d="M237.417 86.655l-30.226 52.352-30.224-52.352h-48.303L207.19 222.67l78.53-136.015z" fill="#435466"/></svg>
```
Comment (Haroenv) [inline src/img/projects/instantsearch-vue.svg]: Is that because we don't have a specific logo yet?

File: src/algolia-projects.json [hunk 3]
```diff
@@ -76,6 +76,19 @@
     "ranking": 1500
   },
   {
+    "name": "Vue InstantSearch",
+    "category": "Library",
+    "description": "The Vue.js ultimate toolbox for creating instant-search experiences",
```
Comment (Haroenv) [inline src/algolia-projects.json]: The ultimate Vue.js toolbox for creating instant search experiences

Comment (rayrutjes) [inline src/algolia-projects.json]: do you want to take care of that one @JonasBa ?

Comment (JonasBa) [inline src/algolia-projects.json]: Done ;)

**Pull request merged**